src/components/graph/graph.helper.js

```diff
@@ -42,15 +42,17 @@ const NODE_PROPS_WHITELIST = ['id', 'highlighted', 'x', 'y', 'index', 'vy', 'vx'
  * Wtf is a force? {@link https://github.com/d3/d3-force#forces| here}
  * @param  {number} width - the width of the container area of the graph.
  * @param  {number} height - the height of the container area of the graph.
+ * @param  {number} gravity - the force strength applied to the graph.
  * @returns {Object} returns the simulation instance to be consumed.
  * @memberof Graph/helper
  */
-function _createForceSimulation(width, height) {
+function _createForceSimulation(width, height, gravity) {
     const frx = d3ForceX(width / 2).strength(CONST.FORCE_X);
     const fry = d3ForceY(height / 2).strength(CONST.FORCE_Y);
+    const forceStrength = gravity || CONST.FORCE_IDEAL_STRENGTH;
```

Comment:
Do we need the fallback `|| CONST.FORCE_IDEAL_STRENGTH;` won't gravity always be defined?